Core/include/Acts/Surfaces/Surface.hpp

```diff
@@ -420,20 +420,21 @@ class Surface : public virtual GeometryObject,
   /// Return properly formatted class name
   virtual std::string name() const = 0;
 
-  /// Return a Polyhedron for this object
+  /// Return a Polyhedron for surface objects
   ///
   /// @param gctx The current geometry context object, e.g. alignment
-  /// @param lseg Number of segments along curved lines, if the lseg
-  /// is set to one, only the corners and the extrema are given,
-  /// otherwise it represents the number of segments for a full 2*M_PI
-  /// circle and is scaled to the relevant sector
+  /// @param quarterSegments The number of segemtns to approximate a 0.5*pi sector,
+  /// which represents a quarter of the full circle
+  ///
+  /// @note In order to symmetrize the code between sectoral and closed cylinders
+  /// in case of closed cylinders, both (-pi, pi) are given as separate vertices
```

Comment:
I guess this can be kind of glitchy in visualization, but probably not a big deal.

Reply:
I checked on the obj, no effect - and it implies the code quite a bit as I do not have to branch between closed and open cylinders.